app/controllers/application_controller.rb

```diff
@@ -51,6 +57,12 @@ def set_seen_cookie_message
     cookies[:seen_cookie_message] = { value: 'yes', expires: 1.year.from_now, httponly: true }
   end
 
+  def set_cors_headers
+    headers['Access-Control-Allow-Origin'] = '*'
```

Comment:
Possibly - but it's open data which means everyone has access, plus they could just proxy requests through their own server anyway.